pkg/agent/util/net_windows.go

```diff
@@ -147,6 +160,15 @@ func CreateHNSNetwork(hnsNetName string, subnetCIDR *net.IPNet, nodeIP *net.IPNe
 	return hnsNet, nil
 }
 
+func DeleteHNSNetwork(hnsNetName string) error {
+	hnsNet, err := hcsshim.GetHNSNetworkByName(hnsNetName)
+	if err != nil {
```

Comment:
Do you need to check `notFoundError` here?

Reply:
Done